Classifier_Klasse.py

```diff
@@ -7,22 +7,41 @@
 """
 
 import pandas as pd
+import click
 from feature_class_punctuation import PunctFeatures
 from Word_features import count_stopwords
+from exceptions import FeatureException
+from exceptions import InputException
+import warnings
 
 class Classifier():
 
     def __init__(self, df):
         self.df = df
 
-    def train_model(self):
+    def train_model(self, features_for_train):
+        list_user_features = []
+        for c in features_for_train:
+            if c not in list(self.df.columns)[1:]:
+                raise FeatureException(str(c) + " feature is not available")
```

Comment:
Statt der String Konkatenation bieten sich für so Fälle f-Strings an:
`f"{c} feature is not available."`